### PR TITLE
feature: Add support for changing the store payment method field visibility

### PR DESCRIPTION
**Pull request by eduardourso**
## Summary
There was no support for changing the visibility of the store payment method switch after the CardComponent was presented. 

Internally there was a need to be able to hide/show the store payment method field based on the card brand/type.

162455160-9b2b518b-0a10-40f8-a799-31028b11fec7

### CODE REVIEW

**Comment by mohammedDehairy** [comment]
@eduardourso Thanks so much for your contribution!

please make the pull request from develop branch, you seem to have made the pull request from master which is way behind develop, so we can't merge this into develop.

File: Adyen/UI/Form/Items/Toggle/FormTogglechItem.swift [hunk 1]
```diff
@@ -8,7 +8,10 @@ import Foundation
```
Comment (mohammedDehairy) [inline Adyen/UI/Form/Items/Toggle/FormTogglechItem.swift:8]: @eduardourso Thanks so much for your contribution!

please make the pull request from develop branch, you seem to have made the pull request from master which is way behind develop, so we can't merge this into develop.

Comment (mohammedDehairy) [inline Adyen/UI/Form/Items/Toggle/FormTogglechItem.swift:8]: @eduardourso Thanks so much for your contribution!

please make the pull request from develop branch, you seem to have made the pull request from master which is way behind develop, so we can't merge this into develop.

**Review by mohammedDehairy — CHANGES_REQUESTED** [review]

File: Adyen/UI/Form/Items/Toggle/FormTogglechItem.swift (hunk 1 above)
Comment (eduardourso) [reply Adyen/UI/Form/Items/Toggle/FormTogglechItem.swift:8]: Sure, I'll close the PR and opened a new one: https://github.com/Adyen/adyen-ios/pull/852

**Pull request closed without merging**